docs/README.md

```diff
@@ -126,7 +126,7 @@ or
 }
 ```
 
-> ⚠️ *When set as `skip`, the `simulationBehavior` prop defines that the search data should only be fetched using the store's Cache. Caution: In practice, **this may impact the content displayed on store pages, since the cache storage changes according to user interaction in each page**. In order to understand this prop behavior, take a look at the table below!
+> ⚠️ \*When set as `skip`, the `simulationBehavior` prop defines that the search data should only be fetched using the store's Cache. Caution: In practice, **this may impact the content displayed on store pages, since the cache storage changes according to user interaction in each page**. In order to understand this prop behavior, take a look at the table below!
```

Comment:
This back slash should be here?

Reply:
I used @vtex/prettier-config to format the file. This added it automatically, I bet that there's a rule that adds a backlash when it encounters an asterisk; maybe it thinks it's a comment?

Reply:
[Indeed!](https://github.com/prettier/prettier/issues/3837)